tests/fixture.rs

```diff
@@ -3,13 +3,15 @@ use std::path::PathBuf;
 pub enum Fixture {
     CFF,
     TTF,
+    MATH,
 }
 
 impl Fixture {
     pub fn path(&self) -> PathBuf {
         match *self {
             Fixture::CFF => "tests/fixtures/SourceSerifPro-Regular.otf",
             Fixture::TTF => "tests/fixtures/OpenSans-Italic.ttf",
+            Fixture::MATH => "tests/fixtures/latinmodern-math.otf",
```

Comment:
Why do we need the other file, `xits-math.otf`? Can you rename `latinmodern-math.otf` to `LatinModern-Math.otf`?